web/static/css/styles.css

```diff
@@ -136,38 +136,28 @@ header {
   margin-bottom: 40px;
 }
 
-.cagov-step-list.sm {
-  padding-top: 8px;
+.vertical-tracker > li:has(~ .current) .dot-line-inner:before,
```

Comment:
Taking name suggestions for this component. Then I can rename the parent class (currently `vertical-tracker`) and the file name (currently `form-progress`) to the same uniform name!

Vertical tracker comes from here https://template.webstandards.ca.gov/patterns/progress-tracker.html#Vertical-tracker - though it differs in that the colored bubbles show you all your completed and current steps. Step indicator is from USWDS: https://designsystem.digital.gov/components/step-indicator/

Reply:
Thanks for the links explaining the usage of `:has` and `~` to select the previous siblings!